packages/api/src/channels/channels.controller.ts

```diff
@@ -150,15 +150,15 @@ export class ChannelsController {
       const { isValid } = signatureVerify(JSON.stringify(message), signature, jsChannel.ownerMember.controllerAccount)
 
       // Ensure that the signature is valid and the message is not a playback message
-      if (!isValid || new Date(channel.lastActedAt) >= message.timestamp) {
+      if (!isValid || channel.lastActedAt >= message.timestamp) {
         throw new Error('Invalid request signature or playback message. Permission denied.')
       }
 
       // update channel ingestion status
       await this.channelsService.save({
         ...channel,
         shouldBeIngested: message.shouldBeIngested,
-        lastActedAt: message.timestamp.getTime(),
+        lastActedAt: message.timestamp,
```

Comment:
```suggestion
        lastActedAt: message.timestamp,
        ...(message.videoCategoryId ? { videoCategoryId: message.videoCategoryId } : {}),
```
Let's add the possibility of updating the category id.

Reply:
Addressed in [7e29921](https://github.com/Joystream/youtube-synch/pull/118/commits/7e29921c7b62fd1eb8b8b204da3a7fa9a4b3ac15)